crypto/sign.go

```diff
@@ -104,13 +104,13 @@ func SignatureFormatCheck(algo SigningAlgorithm, s Signature) (bool, error) {
 
 // GeneratePrivateKey generates a private key of the algorithm using the entropy of the given seed.
 //
+// The seed minimum length is 32 bytes and it should have enough entropy.
 // It is recommended to use a secure crypto RNG to generate the seed.
-// The seed must have enough entropy and should be sampled uniformly at random.
 //
 // The function returns:
 //   - (false, invalidInputsErrors) if the signing algorithm is not supported or
 //
-// if the seed length is not valid (too short or too long)
+// if the seed length is not valid (less than 32 bytes or larger than 256 bytes)
```

Comment:
I feel like the formatting is a bit messed up here
```suggestion
//   - (false, invalidInputsErrors) if the signing algorithm is not supported or
//     the seed's length is invalid (less than 32 bytes or larger than 256 bytes)

```